File: Java/IfStatements2/GenderGame/GenderGame.java
//IfStatements2 GenderGame 50pts Nico Liao
import java.util.Scanner;
public class GenderGame {
    public static void main(String[] args) {
        Scanner keyboard = new Scanner(System.in);
        String fname, lname, gender, married;
        int age;
        System.out.println("What's your first name");
        fname = keyboard.next();
        System.out.println("What's your last name");
        lname = keyboard.next();
        System.out.println("What's your gender (F or M)");
        gender = keyboard.next();
        if (gender.equalsIgnoreCase("M")) {
            System.out.println("What's your age");
            age = keyboard.nextInt();
            if (age < 20) {
                System.out.println("Then I shall call you " + fname + " " + lname);
            }else if (age >= 20) {
                System.out.println("Then I shall call you Mr." + lname);
            }
        }else if (gender.equalsIgnoreCase("F")) {
            System.out.println("What's your age");
            age = keyboard.nextInt();
            if (age < 20) {
                System.out.println("Then I shall call you " + fname + " " + lname);
            }else if (age >= 20) {
                System.out.println("Are you married (y or n)");
                married = keyboard.next();
                if (married.equalsIgnoreCase("y")) {
                    System.out.println("Then I shall call you Mrs." + lname);
                }else if (married.equalsIgnoreCase("n")) {
                    System.out.println("Then I shall call you Ms." + lname);
                }
            }
        }
    }
}

File: C++/Conversion2/GenderGame/GenderGame.cpp
//IfStatement GenderGame 50pts   Nico Liao
#include <iostream>
using namespace std;
int main()
{
    string fname, lname, gender, married;
    int age;
    cout << "What's your first name" << endl;
    cin >> fname;
    cout << "What's your last name" << endl;
    cin >> lname;
    cout << "What's your gender (F or M)" << endl;
    cin >> gender;
    if (gender == "M")
    {
        cout << "What's your age" << endl;
        cin >> age;
        if (age < 20)
        {
            cout << "Then I shall call you " << fname << " " << lname;
        }
        else if (age >= 20)
        {
            cout << "Then I shall call you Mr." << fname << " " << lname;
        }
    }
    else if (gender == "F")
    {
        cout << "What's your age" << endl;
        cin >> age;
        if (age < 20)
        {
            cout << "Then I shall call you " << fname << " " << lname;
        }
        else if (age >= 20)
        {
            cout << "Are you married (y or n)" << endl;
            cin >> married;
            if (married == "y")
            {
                cout << "Then I shall call you Mrs." << lname << endl;
            }
            else if (married == "n")
            {
                cout << "Then I shall call you Ms." << lname << endl;
            }
        }
    }
    return 0;
}